pymeasure/instruments/oxfordinstruments/itc503.py

```diff
@@ -177,10 +206,14 @@ def __init__(self, resourceName, clear_buffer=True, **kwargs):
         if clear_buffer:
             self.adapter.connection.clear()
 
+        self._T_RANGE[0] = min_temperature
+        self._T_RANGE[1] = max_temperature
```

Comment:
Have you confirmed that this works correctly? Keep in mind that `Instrument.control` is a property *factory*, which iiuc creates the property once at instantiation time. I am not sure if later changes (e.g. during `__init__` or even later) to `self._T_RANGE` have an effect on the property.

Reply:
This seems to work so far; the fact that a mutable (list) is used seems to ensure that the property can be used like this. However, I would agree that this is not the cleanest solution, but didn't see another way. If you have a suggestion, I'd be happy to change it.

Reply:
No other suggestion, I was just surprised that this works at all! But indeed, the mutability seems to be the saving grace here, this will come in handy in other instruments. I think this is quite elegant.